src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@model DirectorySearchResultViewModel
 <li class="navcard navcard--white navcard--border-top navcard--force-border-top navcard--width-100">
     <div class="navcard__content">
-        <a href=@Url.ActionLink("DirectoryEntry", "Directory", new { directorySlug=@Model.DirectorySlug, entrySlug=@Model.Entry.Slug }) data-cy="nav-card-link" class="navcard__link navcard__link--border ">
+        <a href=@Url.ActionLink("DirectoryEntry", "Directory", new { directorySlug=@Model.DirectorySlug, entrySlug=@Model.Entry.Slug }, Context.Request.Scheme, null, null) data-cy="nav-card-link" class="navcard__link navcard__link--border ">
```

Comment:
Is this to fix the "https" requests? Could it be done using @Url.Action?

Reply:
Yes , I think you would have to do the same whether it was Url.Action or Url.ActionLink - the only different being Url.Action would also build the <a> tag not just give you the href value?